.golangci.yml

```diff
@@ -1,110 +1,103 @@
+        - name: empty-block
+        - name: error-naming
+        - name: error-return
+        - name: error-strings
+        - name: errorf
+        - name: exported
+          arguments:
+            - checkPrivateReceivers
+            - sayRepetitiveInsteadOfStutters
+        - name: increment-decrement
+        - name: indent-error-flow
+        - name: range
+        - name: receiver-naming
+        - name: redefines-builtin-id
+        - name: superfluous-else
+        - name: time-naming
+        - name: unreachable-code
+        - name: var-declaration
+    staticcheck:
+      checks:
+        - all
+    testifylint:
+      enable-all: true
+  exclusions:
+    generated: lax
+    presets:
+      - common-false-positives
+      - legacy
+      - std-error-handling
     rules:
-      - name: blank-imports
-      - name: context-as-argument
-        arguments: ["allowTypesBefore": "*testing.T"]
-      - name: context-keys-type
-      - name: dot-imports
-      - name: empty-block
-      - name: error-naming
-      - name: error-return
-      - name: error-strings
-      - name: errorf
-      - name: exported
-        arguments: ["checkPrivateReceivers", "sayRepetitiveInsteadOfStutters"]
-      - name: increment-decrement
-      - name: indent-error-flow
-      - name: range
-      - name: receiver-naming
-      - name: redefines-builtin-id
-      - name: superfluous-else
-      - name: time-naming
-      - name: unreachable-code
-      - name: var-declaration
-      # These ones are usually enabled by default
-      # but enabling them would require some work and further checks
-      # - name: unexported-return
-      # - name: unused-parameter
-      # - name: var-naming
-
-  gocritic:
-    enable-all: true
-    disabled-checks:
-      - appendAssign    # this one is enabled by default usually, but it reports something that needs to be checked
-      - hugeParam       # this one could be considered later
-      - rangeValCopy    # this one should be considered later, but it requires some checks
-      - regexpSimplify  # this one should be considered later, but it requires some checks
-
-  misspell:
-    locale: "US"
-
-  govet:
-    enable-all: true
-    disable:
-      - shadow
-      - fieldalignment
-
-  gofumpt:
-    extra-rules: true
-  staticcheck:
-    checks: ["all"]
-  gci:
-    sections:
-      # Standard section: captures all standard packages.
-      - standard
-      # Default section: catchall that is not standard or custom
-      - default
-      # linters that related to fuego, so they should be separated
-      - localmodule
-
-  testifylint:
-    enable-all: true
+      - linters:
+          - errcheck
+          - gocritic
+          - gosec
+          - govet
+          - ineffassign
+          - misspell
+          - revive
+          - staticcheck
+          - unused
+          - usestdlibvars
+          - whitespace
+        path: _test\.go$
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
I feel like none of these are needed, they were simply added by golangci-lint migrate because they were in golangci-lint v1 default exclusions.

examples should be linted too, but this could be done in a separate PR once this one is merged

Reply:
Interesting, thanks for the info